src/workers/race_event_status_worker.rs

```diff
@@ -334,259 +577,24 @@ fn get_season_race_info(
             Bracket::as_select(),
         ))
         .filter(schema::seasons::dsl::state.eq(season_started_state))
-        .load(db)?
-        .into_iter()
-        .map(|(race, bri, bracket)| RaceInfoBundle { race, bri, bracket })
-        .collect::<Vec<_>>();
+        .load(conn)?;
 
     let players = Player::by_id(
         Some(
             races
                 .iter()
-                .map(|i| [i.race.player_1_id, i.race.player_2_id])
+                .map(|(race, _, _)| [race.player_1_id, race.player_2_id])
                 .flatten()
                 .collect::<_>(),
         ),
-        db,
+        conn,
     )?;
 
-    Ok((races, players))
-}
-
-#[cfg(test)]
-mod tests {
```

Comment:
i feel bad deleting these tests but i kind of hate maintaining unit tests in rust? i wrote these mostly in a spirit of exploration to see what the code i was writing did, and as the code under test drifts, unit tests break very rapidly. i'm used to python where unit tests are easier to write and have more value, and i find them a lot less worthwhile in rust